jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb

```diff
@@ -0,0 +1,730 @@
+    "                        .setKernelSize(new Shape(5, 5))\n",
+    "                        .optPadding(new Shape(2, 2))\n",
+    "                        .setFilters(256).build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Pool.maxPool2DBlock(new Shape(3, 3), new Shape(2, 2)))\n",
+    "        // Use three successive convolutional layers and a smaller convolution\n",
+    "        // window. Except for the final convolutional layer, the number of\n",
+    "        // output channels is further increased. Pooling layers are not used to\n",
+    "        // reduce the height and width of input after the first two\n",
+    "        // convolutional layers\n",
+    "                .add(Conv2D.builder()\n",
+    "                        .setKernelSize(new Shape(3, 3))\n",
+    "                        .optPadding(new Shape(1, 1))\n",
+    "                        .setFilters(384).build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Conv2D.builder()\n",
+    "                        .setKernelSize(new Shape(3, 3))\n",
+    "                        .optPadding(new Shape(1, 1))\n",
+    "                        .setFilters(384).build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Conv2D.builder()\n",
+    "                        .setKernelSize(new Shape(3, 3))\n",
+    "                        .optPadding(new Shape(1, 1))\n",
+    "                        .setFilters(256).build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Pool.maxPool2DBlock(new Shape(3, 3), new Shape(2, 2)))\n",
+    "                .add(Blocks.batchFlattenBlock())\n",
+    "        // Here, the number of outputs of the fully connected layer is several\n",
+    "        // times larger than that in LeNet. Use the dropout layer to mitigate\n",
+    "        // overfitting\n",
+    "                .add(Dropout\n",
+    "                        .builder()\n",
+    "                        .optRate(0.5f)\n",
+    "                        .build())\n",
+    "                .add(Linear\n",
+    "                        .builder()\n",
+    "                        .setUnits(4096)\n",
+    "                        .build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Dropout\n",
+    "                        .builder()\n",
+    "                        .optRate(0.5f)\n",
+    "                        .build())\n",
+    "                .add(Linear\n",
+    "                        .builder()\n",
+    "                        .setUnits(4096)\n",
+    "                        .build())\n",
+    "                .add(Activation::relu)\n",
+    "        // Output layer. Since we are using Fashion-MNIST, the number of\n",
+    "        // classes is 10, instead of 1000 as in the paper\n",
+    "                .add(Linear.builder().setUnits(10).build());\n",
+    "\n",
+    "float lr = 0.01f;\n",
```

Comment:
Can you put this in a different code block?

Reply:
I would want to keep the Model initialization in the same code block then. The reason is because since the block isn't initialized, it then gives a null pointer exception when I run the cell. In order to avoid that ambiguity and problem, I am planning to keep this part:
`
Model model = Model.newInstance("cnn");
model.setBlock(block);
`

in the same cell.